composeCore/src/main/java/com/hamdan/forzenbook/compose/core/composables/Composables.kt

```diff
@@ -128,7 +128,7 @@ fun ErrorText(error: String) {
 }
 
 @Composable
-fun SubmitButton(onSubmission: () -> Unit, label: String, enabled: Boolean) {
+fun SubmitButton(label: String, enabled: Boolean, onSubmission: () -> Unit,) {
```

Comment:
All composables you make should have a modifier it brings in

Reply:
I've added it to the reusable composables, should I also add it to the specific case composables like the Post page specific content composables

Reply:
Not if it's specific like that

Reply:
ok thats what I was thinking as well.